news-explorer-frontend/src/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,23 @@
+import './SearchForm.css';
+import { useRef } from 'react';
+
+function SearchForm(props) {
+  const keywordInput = useRef();
+
+  const handleSearch = (e) => {
+    const keyword = keywordInput.current.value;
+    props.setShowPreloaderClass('');
+    props.onSearch(keyword);
+
+    e.preventDefault();
+  };
+
+  return (
+    <form className='header__form'>
+      <input className='header__input' placeholder='Text not entered' ref={keywordInput} />
+      <button className='header__button' onClick={handleSearch}>Search</button>
```

Comment:
`onClick={handleSearch}`

please, delete it from the button

Submit handlers are added only to tags `form` with event `submit` rather than to submit buttons with event `click` because it also automatically handles `Enter` presses